modules/standard/Map.chpl

```diff
@@ -515,7 +515,7 @@ module Map {
      */
     proc addOrSet(in k: keyType, in v: valType) {
       _enter(); defer _leave();
-      var (found, slot) = findAvailableSlot(k);
```

Comment:
What was the previous output for this? Could we add a quick test for `addOrSet` to lock this in?

Reply:
It was an unresolved call error if you attempted to use this method. I'll have to add a new test tomorrow.